changelog/next/features/3869.md

```diff
@@ -0,0 +1,7 @@
+The new `created_at` field tracks a managed pipeline's creation time.
+
+The new `last_modified` field tracks a managed pipeline's last manual
+modification time.
+
+Pipelines that were created before this change will have the creation and
+modification time set to the pipeline manager launch time.
```

Comment:
Paragraphs render as individual changelog entries. Here's a rewording suggestion that fixes that and I think reads a bit better.

```suggestion
`show pipelines` and the `/pipeline` API endpoints now include `created_at` and `last_modified` fields that track the pipeline's creation and last manual modification time, respectively. Pipelines created with older versions of Tenzir will use the start time of the node as their creation time.
```